style/popup.less

```diff
@@ -53,36 +64,42 @@
 			background-image: inline-svg( '../images/close-white.svg' );
 			top: 40px;
```

Comment:
I still think the close button top: 40px is a bit off, the svg is square size,so  top and right position should be the same to looks more correct in general.

Reply:
discussion : Let's keep it and let design check in here.